platformio.ini

```diff
@@ -61,3 +61,16 @@ lib_deps = Wire
            adafruit/Adafruit BusIO@^1.14.1
            adafruit/Adafruit BNO055@^1.6.1
            adafruit/Adafruit Unified Sensor@^1.1.9
+
+[env:feather_m4_hacky_lora_groundstation]
```

Comment:
As we get more boards are we documenting what physical hardware these are somewhere? Maybe we need BOARDS.md with pictures

Reply:
Also, add this to CI in main.yaml

Reply:
alright I added it